src/main/java/at/ac/tuwien/student/sese2017/xp/hotelmanagement/domain/web/form/CustomerSearchCriteria.java

```diff
@@ -0,0 +1,17 @@
+package at.ac.tuwien.student.sese2017.xp.hotelmanagement.domain.web.form;
+
+import lombok.Data;
+
+/**
+ * This is a data object representing the available criteria to search for customers.
+ * 
+ * @author Michael
+ *
+ */
+@Data
+public class CustomerSearchCriteria {
```

Comment:
Brauchen wir ein Entity objekt um einen String zu wrappen?

Reply:
Ich hab kein Beispiel gefunden in dem in einer Form nur ein String gepostet wurde, immer nur eine ganze Entity. Das is aus Unwissenheit entstanden. Bitte Verbesserungsvorschlag posten wenn man irgendwie direkt den String in den Controller als Suchtext reinbringen kann.